docs/tutorials/sampler.ipynb

```diff
@@ -0,0 +1,211 @@
+{
+ "cells": [
```

Comment:
Thinking about this more I'm a bit confused about the difference between this code and the existing tutorial code we used to invoke the sampler program

```python
# Specify the program inputs here.
program_inputs = {"circuits": qc}

# Specify the backend name.
options = {"backend_name": "ibmq_qasm_simulator"}

job = service.run(
    program_id="sampler",
    options=options,
    inputs=program_inputs,
)
```

Is there a conceptual difference between the sampler program and a sampler primitive? How do both approaches relate to each other?